src/adapter/cloudflare-pages/handler.test.ts

```diff
@@ -2,30 +2,61 @@ import { getCookie } from '../../helper/cookie'
 import { Hono } from '../../hono'
 import { HTTPException } from '../../http-exception'
 import type { EventContext } from './handler'
-import { handle, handleMiddleware } from './handler'
+import { handle, handleMiddleware, serveStatic } from './handler'
 
 type Env = {
   Bindings: {
     TOKEN: string
-    eventContext: EventContext
+  }
+}
+
+function createEventContext(
+  context: Partial<EventContext<Env['Bindings']>>
+): EventContext<Env['Bindings']> {
+  return {
+    data: {},
+    env: {
+      ...context.env,
+      ASSETS: { fetch: vi.fn(), ...context.env?.ASSETS },
+      TOKEN: context.env?.TOKEN ?? 'HONOISCOOL',
+    },
+    functionPath: '_worker.js',
+    next: vi.fn(),
+    params: {},
+    passThroughOnException: vi.fn(),
+    request: new Request('http://localhost/api/foo'),
+    waitUntil: vi.fn(),
+    ...context,
   }
 }
 
 describe('Adapter for Cloudflare Pages', () => {
   it('Should return 200 response', async () => {
     const request = new Request('http://localhost/api/foo')
     const env = {
+      ASSETS: { fetch },
       TOKEN: 'HONOISCOOL',
     }
+    const waitUntil = vi.fn()
+    const passThroughOnException = vi.fn()
+    const eventContext = createEventContext({
+      request,
+      env,
+      waitUntil,
+      passThroughOnException,
+    })
     const app = new Hono<Env>()
+    const appFetchSpy = vi.spyOn(app, 'fetch')
     app.get('/api/foo', (c) => {
-      const reqInEventContext = c.env.eventContext.request
-      return c.json({ TOKEN: c.env.TOKEN, requestURL: reqInEventContext.url })
```

Comment:
From my understanding, this was ensuring that the original `eventContext` was added to `c.env`

I replaced this by adding an assertion that `vi.spyOn(app, 'fetch')` was called with `{ ...env, eventContext }`